libs/domain/product/src/services/adapter/normalizers/category-id/category-id.normalizer.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProductModel } from '../../../../models';
+import { categoryIdNormalizer } from './category-id.normalizer';
+
+describe('Product Node Normalizer', () => {
+  it('should transform ApiProductModel.CategoryNodes[] into DeserializedNode', () => {
+    const mockTransformed = [
+      { isActive: false, nodeId: 4 },
+      { isActive: true, nodeId: 8 },
+    ] as ApiProductModel.CategoryNodes[];
+    const normalized = categoryIdNormalizer(mockTransformed);
+
+    expect(normalized).toEqual({
+      categoryId: String(mockTransformed[1].nodeId),
+    });
+  });
```

Comment:
I'd bring all category Ids to the client model, as we might want to use them all in other stories.